packages/modular-scripts/craco.config.js

```diff
@@ -85,6 +85,9 @@ module.exports = {
         testMatch: ['<rootDir>/*/src/**/*.{spec,test}.{js,ts,tsx}'],
         coverageDirectory: path.resolve(modularRoot, 'coverage'),
         collectCoverageFrom: ['<rootDir>/*/src/**/*.{js,ts,tsx}', '!**/*.d.ts'],
+        setupFiles: jestConfig.setupFiles.concat([
+          require.resolve('./jest-setupFiles.js'),
```

Comment:
My personal preference would be `path.join(__dirname, ...)`, but I guess the semantics of `require` are pretty baked in now.

Reply:
I don't recollect why I did require.resolve here. Will change.